spring-integration-amqp/src/main/java/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.java

```diff
@@ -59,7 +59,7 @@
  */
 public class DefaultAmqpHeaderMapper extends AbstractHeaderMapper<MessageProperties> implements AmqpHeaderMapper {
 
-	static final boolean CONSUMER_METADATA_PRESENT;
+	private static final boolean CONSUMER_METADATA_PRESENT;
```

Comment:
:+1: 
Ideal!

Now only need to find time, hands and, of course, mind to catch up all the work there as well :smile: